### ISSUE TITLE
Zero-valued noise level prediction

### ISSUE THREAD

**Issue by antonveselov** [IDKiro/CBDNet-pytorch#6]
Hello.

First of all, thank you for making such a good work of providing a pytorch implementation of CBDNet. However, I noticed some strange behaviour of noise level prediction sub-network. I downloaded the datasests and run training with default parameters. While training I also dump the predicted noise map and in the middle of the first epoch I see that the predicted noise in most cases consists of zeros only. I tried modifying the weights in loss but it does not help. Actually, what I see is that the network without noise prediction tends to work better. Could you, please, give any comments on this? Am I doing something wrong?

Thank you.

**Comment by IDKiro** [comment]
i am sorry that i am very busy now. if you can train for more epochs first, maybe you can get the different result.
